application/modules/front/views/collection/printhadith.php

```diff
@@ -18,8 +18,8 @@
 	echo "<!-- Begin hadith -->\n\n";
 	echo "<a name=$linknum></a>\n";
 	
+			echo "<div class=\"hadith_reference_sticky\">";
```

Comment:
This  should be inside the div. We only want to display a sticky reference when the reference has been verified (status = 4)

Reply:
The reason I put this outside the `if` block was because I wanted the sticky reference to show in all cases for all Ḥadīth. This would make the look consistent. Previously, it was only showing for the Ḥadīth with standard numbering, but not for the other types of Ḥadīth. Nevertheless, we do have an alternative numbering for every Ḥadīth, so my thinking was that why not show that alternative numbering for non-standard-numbered Ḥadīth? Why leave the numbering altogether in this case? We are using that numbering at the bottom of the Ḥadīth, so why hide it from the top? It is a proper numbering after all. This was my thinking that was instigated by the inconsistent display for the two types of Ḥadīth.

Nevertheless, if the business logic, which you decide, dictates that we should not show sticky reference at the top of non-standard numbered Ḥadīth on mobile, then I will update that. Some CSS will also need to be changed to account for empty space left by the sticky reference in my design.

Reply:
If the book status is not 4, we haven't verified the reference and it very likely doesn't conform to any printed edition. In that case I would not like for us to display that number prominently.